internal/impl/generate.go

```diff
@@ -136,3 +140,24 @@ func makeFlakeFile(outPath string, plan *plansdk.ShellPlan) error {
 	}
 	return nil
 }
+
+func isProjectInGitRepo(dir string) bool {
```

Comment:
Didn't you implement something similar to check if the directory contains `.git` and only start devbox cloud shell when `.git` is present? Can we share the code between them or no?

Reply:
That sounds similar, but in practice is fairly different:
https://github.com/jetpack-io/devbox/blob/987d51874e446fde6ef13dc751c6babafb356efc/internal/cloud/cloud.go#L480